datasets/fr/tresor/crawler.py

```diff
@@ -8,6 +8,11 @@
 from zavod import Context, Entity
 from zavod import helpers as h
 
+HEADERS = {
+    "Accept": "application/json",
+    "User-Agent": "custom-agent",
```

Comment:
It's nice to include opensanctions.org in the user agent if we can make it work with that text in there. That way if we hurt them they can get in touch with us easily.

Reply:
It doesn't work unfortunately in this case